src/components/shapes/display_labels.js

```diff
@@ -0,0 +1,268 @@
+'use strict';
```

Comment:
This is just the `drawLabels()` function which used to be in `shapes/draw.js` being moved into its own file. No code changes.